### ISSUE TITLE
Join Us page: Add anchors to sections Advise Us, Volunteer with Us, and Partner with Us

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#3026]
### Overview
We need to be able to go directly to the Advise Us, Volunteer with Us, and Partner with Us sections on the [Join Us page](https://www.hackforla.org/join) so that we can link to those sections from the header and footer menus.  For this issue, we will add anchors to those sections.

### Action Items
In [pages/join-us.html](https://github.com/hackforla/website/blob/gh-pages/pages/join-us.html),
- [x] Add an anchor to the Advise Us section so that https://www.hackforla.org/join#advise will jump to that section. For example, adding the line of code `<a class="anchor" id="advise"></a>` before
https://github.com/hackforla/website/blob/d40336f2195c6256c6d8d6aa6cdd5f1731a9bd3e/pages/join-us.html#L28
so that it becomes
  ```
    <a class="anchor" id="advise"></a>
    <div class="page-card card-primary page-card-lg page-card--join page-card--large-icon-container">
  ```
- [x] Add an anchor to the Volunteer with Us section so that https://www.hackforla.org/join#volunteer will jump to that section
- [x] Add an anchor to the Partner with Us section so that https://www.hackforla.org/join#partner will jump to that section
- [x] Using Docker, check that each anchor link jumps to its corresponding section. For example, if I wanted to check https://www.hackforla.org/join#advise will jump to the Advise Us section, I would need to use the address localhost:4000/join#advise
- [ ] Once the pull request associated with this issue is approved and merged, 
  - [x] Please update and edit issue https://github.com/hackforla/website/issues/2393 by
    - [x] Checking off the dependency for this issue
    - [ ] If all dependencies are checked off, please move issue https://github.com/hackforla/website/issues/2393 to the New Issue Approval column and remove the Dependency label
  - [x] Please update and edit issue #2455 by
    - [x] Checking off the dependency for this issue
    - [ ] If all dependencies are checked off, please move issue #2455 to the New Issue Approval column and remove the Dependency label 

### Resources/Instructions
- [Join Us page](https://www.hackforla.org/join)
- [pages/join-us.html](https://github.com/hackforla/website/blob/gh-pages/pages/join-us.html)
- Example page that has anchors: https://www.hackforla.org/about
- Tracked in https://github.com/hackforla/website/issues/2481#issuecomment-968349924
- Related to issues
  - https://github.com/hackforla/website/issues/2393
  - https://github.com/hackforla/website/issues/2455

**Comment by scorbz9** [comment]
Hi @Pauljsyi! Just FYI I moved this one to the In progress column as you appear to still be currently assigned to it.

**Comment by SAUMILDHANKAR** [comment]
@Pauljsyi Please add your ETA and availability for this issue. Thank you.

**Comment by Sparky-code** [comment]
@Pauljsyi Checking in to see if you are still working on this issue or if your availability has changed. Please add your ETA and availability. Thanks!

**Comment by Sparky-code** [comment]
@Pauljsyi I am going to unassign you from this issue as I have not seen any updates from you

**Comment by jdingeman** [comment]
Availability for this week: 4 hours
My estimated ETA for completing this issue: EOD 7/21/2022
